app/assets/stylesheets/application.scss

```diff
@@ -23,6 +23,7 @@ $govuk-use-legacy-palette: false;
 @import 'govuk_publishing_components/components/_layout-header';
 @import 'govuk_publishing_components/components/_layout-footer';
 @import 'govuk_publishing_components/components/_title';
+@import 'govuk_publishing_components/components/_search';
```

Comment:
Isn't this change already in https://github.com/alphagov/datagovuk_find/pull/779/files?

Reply:
Indeed it is. Because both these PR's required very similar changes, I had a plan to merge one and then rebase the other. Whichever PR get's merged first is the one that introduced the change. I wanted to avoid making a separate PR for just this, mainly if I'm honest because I'm finding it difficult to get review on these datagovuk PRs.